layouts/layout-basics/src/exercises/exercise2/positioning.css

```diff
@@ -0,0 +1,75 @@
+/* 
+  📚 Learn CSS - the Box Model 📚
```

Comment:
Can you number each CSS file with the exercise it corresponds to? There should be something on `Exercise 2` here.